### PR TITLE
Portfolio: desktop version

### PR DESCRIPTION
**Pull request by jcmunav63**
**Description**

For the fourth project in your portfolio website, you will:

- Use media queries to make sure that your website displays correctly on desktop screens.

You have time for this project scheduled in to 2 days. Please ask for a code review when you are ready.

IMPORTANT NOTE: Read all requirements before you start building your project.

**General requirements**
- Make sure that there are [no linter errors](https://github.com/microverseinc/linters-config).
- Make sure that you used correct ([Github flow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/github_flow.md).
- Make sure that you documented your work [in a professional way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md).

**HTML/CSS requirements**
- Follow our list of [best practices for HTML & CSS](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md).

**Project requirements**
IMPORTANT NOTE: For each Microverse project, you need to create a separate pull request. [Read how to work with multiple pull requests in your one repository in the most efficient way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/multiple_pull_requests.md).

- You need to make sure that your dekstop version looks like the one in the Figma design template that you chose. We include a screenshot of Template 1 below for reference, but you should use the template that you chose.
- IMPORTANT NOTE some Figma design templates include two input fields (for first name and last name) in the desktop version and one input field (for full name) in the mobile version (look for those cases in Contact me or Get in touch sections). Please ignore this part of the design template and use only one input for the full name for all versions.
- Use media queries in your CSS file to make your desktop version look like the template you selected (Desktop: >=768px).
- In order to lay out the elements on the page you should use Grid and Flexbox.
- You don't need to implement any functionality that requires JavaScript.
- You must replicate the design as much as possible (e.g., font, colors, images, text, margins) using [the info from the Figma templates](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

### CODE REVIEW

File: css/styles.css [hunk 1]
```diff
@@ -1,535 +1,1148 @@
...
-  font-size: 20px;
-  font-weight: 500;
-  line-height: 24px;
-}
+  .second {
+    width: 40%;
+    display: flex;
+    flex-direction: column;
+    padding-left: 15px;
+  }
```
Comment (TadesseDev) [inline css/styles.css]: - [ ] Dear @jcmunav63, For the `about me` section. the expected behavior is for the left one to take 40% and the right one to the remaining 60% of the page's width. this makes your design looks like the one in figma. 
- [ ] Please also consider adding the lines under each  `Language`, `Framework`, and `Skill`. This is to make your design the exact clone of the given one.
### Figma design 
image

### Your design 
image

File: index.html [hunk 2]
```diff
@@ -16,9 +16,21 @@
...
+                        <li><a class="poppins nav-menu-text" href="#contact-form">Contact</a></li>
+                    </ul>
+                </div>
+            </div>
+        </nav>
     </header>
     <section class="headline">
-        <h1 class="name poppins">I'm Juan Carlos. Glad to see you!</h1>
+        <h1 id="delay" class="name poppins">I'm Juan Carlos. Glad to see you!</h1>
         <p class="desc poppins">I'm a software developer! I can help you build a product, feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don't hesitate to contact me.</p>
```
Comment (TadesseDev) [inline index.html:34]: - [ ] Please consider adding All the decoration images, currently, your design does not show any decoration images. Remember as a frontend engineers we are expected to deliver almost identical webpages. 
### Your design as of now,
image

File: index.html [hunk 3]
```diff
@@ -175,20 +290,19 @@ <h2 class="about poppins">About Myself</h2>
...
+            <div class="skills">
                 <p class="skill poppins">Skills</p>
                 <button name="button-arrow3" type="button"> <img class="arrow-right" alt="" width="512px" height= "512px" src="images/right-arrow.png"></button>
+            </div>
+            <hr/>
         </div>
-        <hr/>
     </section>
     <footer id="foot">
         <section id="contact-form">
```
Comment (TadesseDev) [inline index.html:308]: - [ ] Please consider adding the background color for the `contact section, currently, your design has a white background which does not transparent with the text color. Remember as a frontend engineers we are expected to deliver almost identical webpages.

### Your design as of now,
image

**Review by TadesseDev — CHANGES_REQUESTED** [review]
### Hi @jcmunav63 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
212094426-b41dd457-c9db-4ab3-bbe9-176dd8dd08b7




## Highlights
 - [x] Good job overall following the template design ✅
 - [x]  Correct GitHub Flow  ✅
 - [x]  Good commit history  ✅
 - [x]  Good PR title/description  ✅
 - [x]  All linters passed  ✅
 
## Required Changes ♻️
- [ ] Please notice the link for your MIT file is broken, Please make sure to add the MIT license file and update its content in regard to your name and project. Also replace the `[YOUR NAME]` part with your name, and `[APP TYPE]` with your app type.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 3 above)
Comment (jcmunav63) [reply index.html:308]: I'm sorry, but my design in my computer is showing both, the background for the heading and the background for the contact form, for both, the mobile and the desktop version. Do you mean another background?

**Review by AdnanOlarmmi — APPROVED** [review]
Hi @jcmunav63,

This is Adnan Is-haq, your beloved code reviewer, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Approved office man

## Highlights
- Nice one fixing the required changes.
- Descriptive Pull Request.
- Cool UI design.

## Optional suggestions
_Check the comments under the review._

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**